Resizetizer.NT/SkiaSharpSvgTools.cs

```diff
@@ -1,116 +1,50 @@
 ﻿using SkiaSharp;
-using Svg;
 using Svg.Skia;
 using System;
-using System.Collections.Generic;
 using System.Diagnostics;
 using System.Drawing;
-using System.IO;
-using System.Text;
-using System.Text.RegularExpressions;
 
 namespace Resizetizer
 {
-	internal class SkiaSharpSvgTools
+	internal class SkiaSharpSvgTools : SkiaSharpTools, IDisposable
```

Comment:
Base type of `SkiaSharpTools` does all the work, we just load, provide the size and draw.